torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16738,6 +16738,10 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         torch_opinfo_name="movedim",
         supports_nvfuser=False,
     ),
+    PythonRefInfo(
```

Comment:
The sample inputs for bucketize are very small: 

https://github.com/pytorch/pytorch/blob/cebf08afb24dec0720935b9a9bd64ecf05b472d5/torch/testing/_internal/common_methods_invocations.py#L2666

What about adding some larger reference inputs to improve test coverage? The original `torch` function will be tested against the Python reference using those.

An example of adding reference inputs is 

https://github.com/pytorch/pytorch/blob/cebf08afb24dec0720935b9a9bd64ecf05b472d5/torch/testing/_internal/common_methods_invocations.py#L1404

Typically reference inputs yield from the sample inputs generator so they are weakly more coverage than sample inputs. The reference inputs don't have to be especially complicated or generated in a very clever way. Maybe several handpicked inputs would be enough?

Reply:
Yes, I added this. I actually was running tests with larger sizes when writing the decomp on my side to make sure it worked correctly, so this makes a lot of sense.